week-2-practise/level-3/3-sort.js

```diff
@@ -4,7 +4,9 @@
 
 var nums = [10, 1, 5, 29, 100];
 var nums2 = [11, 6, 3, 29, 12];
-var sortedNums; // complete this statement
+var sortedNums = nums.concat(nums2).sort(function(a, b) {
```

Comment:
Good stuff. This is correct (and useful to know if you want to use a different sort order), but you can just use `sort()` here without a custom sorting function, since the default behaviour is the same.

Reply:
thanks

Reply:
Actually I'm wrong. Javascript is confusing and sorts numbers as if they are strings by default, so you do need to pass a function to it.